packages/sdk/src/display/mapAttributes.ts

```diff
@@ -0,0 +1,35 @@
+import { DateOnly, TypedArrayEncoder } from '@credo-ts/core'
+import { formatDate, isDateString } from '../utils/date'
+import { detectImageMimeType } from '../utils/image'
+
+type MappedAttributesReturnType =
+  | string
+  | number
+  | boolean
+  | { [key: string]: MappedAttributesReturnType }
+  | null
+  | undefined
+  | Array<MappedAttributesReturnType>
+export function recursivelyMapAttributes(value: unknown): MappedAttributesReturnType {
```

Comment:
I think this is only used for mdoc right? If so we should probably make it clear in the function name